README.md

```diff
@@ -36,6 +36,43 @@ Type the name of [any `iron-*` or `paper-*` element](https://elements.polymer-pr
 </dom-module>
 ```
 
+### [pe2015] polymer element ES2015
+
+```html
+<dom-module id="${2}">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+  </template>
+  <script>
+    (function (Polymer) {
+      'use strict';
+      class ${1} {
+        beforeRegister() {
+          this.is = '${2}';
+
+          this.properties = {
+            ${3}
+          };
+        }
+
+        attached() {
+           this.async(() => {
```

Comment:
Oh I think using `attached` is fine. I was specifically asking about `async` though. It doesn't seem like you should need it. I would remove it from the snippet, otherwise folks will think they _always_ have to call it.